src/Amplitude.Net/AmplitudeForUser.cs

```diff
@@ -4,7 +4,9 @@ namespace Amplitude.Net;
 
 public class AmplitudeForUser: AmplitudeFor
 {
-    public AmplitudeForUser(ILogger logger, IAmplitudeSender amplitudeSender, string userId) : base(logger, amplitudeSender, new KeyValuePair<string, object>("user_id", userId))
+    public AmplitudeForUser(ILogger logger, IAmplitudeSender amplitudeSender, string userId, string? deviceId) : base(logger, amplitudeSender, 
+        new [] {new KeyValuePair<string, object?>("user_id", userId), new KeyValuePair<string, object?>("device_id", deviceId)}
```

Comment:
This could probably be a little less verbose? 
Isn't there some .NET 6+ compiler magic that lets it infer types better?

Reply:
target is netstandard2 compat

Reply:
Also.. who cares. The code consumer doesn't have to deal with it :)